go.mod

```diff
@@ -3,59 +3,59 @@ module github.com/giantswarm/azure-operator/v4
 go 1.14
 
 require (
-	github.com/Azure/azure-sdk-for-go v43.2.0+incompatible
+	github.com/Azure/azure-sdk-for-go v45.0.0+incompatible
 	github.com/Azure/azure-storage-blob-go v0.8.0
-	github.com/Azure/go-autorest/autorest v0.10.2
-	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
-	github.com/Azure/go-autorest/autorest/to v0.3.0
+	github.com/Azure/go-autorest/autorest v0.11.2
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.0
+	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/coreos/go-semver v0.3.0
-	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32 // indirect
-	github.com/giantswarm/apiextensions v0.4.10
+	github.com/giantswarm/apiextensions v0.4.17-0.20200723160042-89aed92d1080
 	github.com/giantswarm/appcatalog v0.1.11
-	github.com/giantswarm/apprclient v0.2.0
+	github.com/giantswarm/apprclient v0.2.1-0.20200724085653-63c7eb430dcf
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/certs v0.2.0
-	github.com/giantswarm/e2e-harness v0.3.0
+	github.com/giantswarm/certs v0.2.1-0.20200805143002-4a70288f05cc
```

Comment:
I just created a hacky branch off of v0.2.0 so we can test.

As discussed I think upgrading to v2 to prepare for v3 will help. cc @tim @nprokopic